### PR TITLE
Add License, update metadata, link to docs

### PR DESCRIPTION
**Pull request by samuelallan72**

### CODE REVIEW

File: src/metadata.yaml [hunk 1]
```diff
@@ -16,7 +16,8 @@
 
 name: sysconfig
 summary: System related configurations
-maintainer: BootStack Charmers <bootstack-charmers@lists.canonical.com>
+maintainers:
+  - BootStack Charmers <bootstack-charmers@lists.canonical.com>
```
Comment (chanchiwai-ray) [inline src/metadata.yaml:20]: @Pjack @aieri any idea what name should we choose

**Review by chanchiwai-ray — COMMENTED** [review]
I think we need to change these but I have no information

File: src/metadata.yaml (hunk 1 above)
Comment (Pjack) [reply src/metadata.yaml:20]: Solutions Engineering <solutions-engineering@lists.canonical.com>

**Review by chanchiwai-ray — DISMISSED** [review]
LGTM

**Review by jneo8 — APPROVED** [review]

**Review by dashmage — APPROVED** [review]

**Review by sudeephb — DISMISSED** [review]

**Comment by rgildein** [comment]
@samuelallan72 can you targeted main instead of master, since master is no longer default branch

**Pull request closed without merging**